### ISSUE TITLE
Task iterations

### ISSUE THREAD

**Issue by w-bonelli** [Computational-Plant-Science/plantit#305]
### Overview

It would be nice to have the ability to submit multiple task instances of a particular workflow to run in parallel. This is supported already if the workflow is configured to consume files in an input directory, but we should also allow this for workflows that don't accept inputs.

### Use case

One potential use case here might be Monte Carlo simulations, with each starting from a unique random seed: such a workflow might not need any input data, but we would still like to be able to submit some large number of runs at once.

### Implementation

We can check in `workflow.vue` if the workflow accepts inputs &mdash; if not, append a spin button on the right side here:

<img width="1051" alt="Screen Shot 2022-05-11 at 11 35 34 AM" src="https://user-images.githubusercontent.com/10502868/167890020-9e12213c-206f-44e5-acae-45bee30036ec.png">

Maybe called `Iterations` or `Repetitions` or `Trials`, defaulting to 1.
